src/views/Auction/AuctionDetails.js

```diff
@@ -1,139 +1,234 @@
 import { Box, Grid, Paper, SvgIcon, Typography, useTheme, Button } from "@material-ui/core";
+import { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useWeb3Context } from "../../hooks/web3Context";
 import { ReactComponent as DotIcon } from "./../../assets/icons/dot.svg";
-import AuctionController from "./AunctionController";
+import { txnButtonText } from "src/slices/PendingTxnsSlice";
+import { claimTokens } from "../../slices/AuctionSlice";
+import AuctionController from "./AuctionController";
 import LineChartAuction from "./LineChartAuction";
+import { durationAsString } from "../../helpers";
 
-const AuctionDetails = ({ auctionStatus }) => {
+const AuctionDetails = ({
+  isLoading,
+  auctionStatus,
+  startPrice,
+  minimumPrice,
+  tokenPrice,
+  totalTokens,
+  onCommitTokens,
+  tokensClaimable,
+  totalTokensCommitted,
+  commitedFrax,
+  fraxBalance,
+  startTime,
+}) => {
   const theme = useTheme();
+  const { address, connect, provider, chainID } = useWeb3Context();
+  const dispatch = useDispatch();
+  const remainingTokensShare = 100 - (totalTokensCommitted / totalTokens) * 100;
+  const [timeLeft, setTimeLeft] = useState(durationAsString(new Date().getTime(), startTime * 1000));
+  const pendingTransactions = useSelector(state => state.pendingTransactions);
 
-  // sample data for line chart
   const data = [
```

Comment:
why `"AUCTION PRICE"` instead of `auctionPrice`?

Reply:
yea, makes sense. pushing updates now